bin/makeMarkerStats.R

```diff
@@ -136,26 +120,61 @@ colnames(counts) <- colData(counts)$Barcode
 # Now read in the markers
 
 print("Loading markers...")
-cluster_marker_files <- list.files(path = opt$cluster_markers_dir, pattern = basename(opt$cluster_markers_file_pattern), full.names = TRUE)
-
-# Order cluster markers by k 
-
-k_vals <- sub('markers_([0-9]+).tsv', '\\1', basename(cluster_marker_files))
-marker_files <- structure(cluster_marker_files[order(as.numeric(k_vals))], names = sort(as.numeric(k_vals)))
-
-# Add in cell markers where provided
+all_marker_files <- unlist(lapply(c('cluster', 'meta'), function(markerType){
+    
+    if (! is.na(opt[[paste0(markerType, '_markers_dir')]])){
+        marker_files <- list.files(path = paste0(markerType, '_markers'), pattern = opt[[paste0(markerType, '_markers_file_pattern')]], full.names = TRUE)
+        
+        if (length(marker_files) > 0){
+
+            marker_cell_groupings <- gsub('_', ' ', sub('markers_(.+).tsv', '\\1', basename(marker_files)))
```

Comment:
not sure what this line is doing?

Reply:
If you have a file name like /path/to/markers_thing.tsv, it's extracting 'thing'.